src/nomad_material_processing/general.py

```diff
@@ -545,24 +572,39 @@ class SubstrateCrystalProperties(CrystalProperties):
             component=ELNComponentEnum.EnumEditQuantity,
         ),
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
         type=str,
-        description="""
-        Alignment of crystal lattice with respect to a vector normal to the surface
-        specified using Miller indices.
-        """,
+        description="""Reciprocal (or Absolute) Miller indices, (hkl),
+        denote a family of planes specified an orthogonal vector in the basis of
+        the primitive reciprocal lattice vectors. """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+            label='Reciprocal Indices (hkl)',
+        ),
+    )
+    hkl_direct = Quantity(
+        type=str,
+        description="""Direct (or Relative) Miller indices, [hkl],
+        denote a family of planes specified an orthogonal vector in the basis of
```

Comment:
Pls double-check: I think that here we are really dealing with a direction in real-space. I'll check the Kittel.

Reply:
From Kittel (popular intro book to solid state):
"Displacement takes place along crystallographic planes with a set of small
Miller indices, such as the {111} planes in fcc metals and the {110}, {112}, and
{123} planes in bcc metals.
The slip direction is in the line of closest atomic packing, <110> in fcc
metals and <111> in bcc metals (Problem 1). "